example_ws/src/example_pkg/package.xml

```diff
@@ -60,7 +60,7 @@
   <exec_depend>rospy</exec_depend>
   <exec_depend>std_msgs</exec_depend>
 
-  <test_depend>python-coverage</test_depend>
+  <test_depend>code_coverage</test_depend>
```

Comment:
Is this dependency met by the clone of the code_coverage repo, which is specified via the upstream.repos file?

Reply:
yes it is (I added python-coverage there, we don't actually directly call python-coverage at all from this repo)